integration-test-standard/src/test/resources/cassandra.yaml

```
@@ -1208,64 +1166,86 @@ gc_warn_threshold_in_ms: 1000
+# Automatically upgrade sstables after upgrade - if there is no ordinary compaction to do, the
+# oldest non-upgraded sstable will get upgraded to the latest version
+# automatic_sstable_upgrade: false
+# Limit the number of concurrent sstable upgrades
+# max_concurrent_automatic_sstable_upgrades: 1
+
+# Audit logging - Logs every incoming CQL command request, authentication to a node. See the docs
+# on audit_logging for full details about the various configuration options.
+audit_logging_options:
+    enabled: false
+    logger:
+      - class_name: BinAuditLogger
```

yamllint

[warning] wrong indentation: expected 8 but found 6

(indentation)